lib/datasource.js

```diff
@@ -451,10 +460,13 @@ DataSource.prototype.defineScopes = function(modelClass, scopes) {
   }
 };
 
-/*!
- * Define relations for the model class from the relations object
- * @param modelClass
- * @param relations
+/**
```

Comment:
IIRC, we intentionally use `/*!` to not expose this method to generated API references.